MAINTAINERS

```diff
@@ -14,3 +14,7 @@ M: Xu Tian <xutian@redhat.com>
 M: Yanbing Du <ydu@redhat.com>
 M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
 M: Hao Liu <hliu@redhat.com>
```

Comment:
Hao Liu and Yanbing Du were not in Red Hat, and didn't do contribution to this project any more. Therefore remove them from this list.